stdpopsim/genetic_maps.py

```diff
@@ -177,4 +177,9 @@ def get_chromosome_map(self, name):
         if not self.is_cached():
             self.download()
         map_file = os.path.join(self.map_cache_dir, self.file_pattern.format(name=name))
-        return msprime.RecombinationMap.read_hapmap(map_file)
+        if(os.path.exists(map_file)):
```

Comment:
Does msprime not already raise a FileNotFound error? Or is a better message you're looking for? This is probably a ValueError I'd say:
```python
if not os.path.exists(map_file):
    raise ValueError("Chromosome map for '{}' not found".format(name)
 return msprime.RecombinationMap.read_hapmap(map_file)
```